src/Makefile

```diff
@@ -57,11 +57,16 @@ deploy-dev:
 .PHONY: deploy-dev
 
 deploy:
-	python deployment/prepare_to_deploy.py --dev
+	python3 deployment/prepare_to_deploy.py --dev
 	docker-compose build
 	docker-compose push
 .PHONY: deploy
 
+.PHONY: migrations
+migrations: 
+	python3 manage.py makemigrations
+	python3 manage.py migrate
+
```

Comment:
adding this so that i can just do `make migrations` and have everything done for me